src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,65 @@
+import {createIntegrationSdkInstance, createIntegrationTestUser, removeIntegrationTestUser} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter', () => {
+  let roomsSDKAdapter, sdk, user;
+
+  // Since these are integration tests with live data,
+  // increase the async "idle" timeout so jest doesn't error early.
+  jest.setTimeout(30000);
+
+  beforeAll(async () => {
+    user = await createIntegrationTestUser();
+    sdk = createIntegrationSdkInstance(user.token);
+    await sdk.internal.device.register();
+    await sdk.internal.mercury.connect();
+  });
+
+  beforeEach(() => {
+    roomsSDKAdapter = new RoomsSDKAdapter(sdk);
+  });
+
+  describe('getRoom() returns', () => {
+    test('a room in a proper shape', async (done) => {
+      const createdRoom = await sdk.rooms.create({title: 'Webex Teams Test Room'});
+      const subscription = roomsSDKAdapter.getRoom(createdRoom.id).subscribe((room) => {
+        expect(room).toMatchObject({
+          ID: createdRoom.id,
+          title: createdRoom.title,
+        });
+        subscription.unsubscribe();
+        done();
+      });
+    });
+
+    test('an updated room title after subscribing', async (done) => {
+      const createdRoom = await sdk.rooms.create({title: 'Webex Teams Test Room'});
+      let hasUpdated = false;
+      const updatedTitle = 'Updated Test Title';
+      const subscription = roomsSDKAdapter.getRoom(createdRoom.id).subscribe(async (room) => {
```

Comment:
not too sure why we have this `isUpdated` check. if we subscribe to it first and then make changes, shouldn't that trigger the expect statement automatically?

Reply:
I'll add some comments, but this is due to the first `next` being the current state of the room and then we get another subscription update after the room title changes. Not sure how else you want me to do it.